apps/actions-dependencies-updater/src/github.mts

```diff
@@ -261,18 +261,24 @@ export async function getActionFile(
   return actionFile;
 }
 
-async function getFileFromGithub(
+export async function getFile(
   octokit: Octokit,
   owner: string,
   repo: string,
   path: string,
   ref: string,
 ) {
   try {
+    if (path.startsWith("/")) path = path.substring(1);
+    if (path.startsWith("./")) path = path.substring(2);
+
+    if (path === "") {
+      log.warn("Empty path provided to github.getFile. Returning empty.");
```

Comment:
Just confirming if this should be an error instead?

Reply:
Upgraded to error - a lot of the levels are all over the place to not overload people who are using the tool.